src/Symfony/Component/HttpKernel/Controller/ControllerResolver.php

```diff
@@ -125,7 +125,11 @@ protected function doGetArguments(Request $request, $controller, array $paramete
         $arguments = array();
         foreach ($parameters as $param) {
             if (array_key_exists($param->name, $attributes)) {
-                $arguments[] = $attributes[$param->name];
+                if ($param->isVariadic() && is_array($attributes[$param->name])) {
```

Comment:
you first have to check if this method exists, as Symfony should also support PHP 5.3, 5.4 and 5.5.